tests/base/entities/test_tissue.py

```diff
@@ -13,7 +13,7 @@ def test_uberon_tissue_inspect_ontology_id():
         ]
     )
 
-    ts = bt_base.Tissue(source="uberon")
+    ts = bt_base.Tissue(source="uberon", version="2024-08-07")
```

Comment:
This should automatically pulls the latest version, so I'd not pass version here to be easier for future.

Reply:
If we do that, the test runs forever. See my comment in the PR. We need the parquet on the instance first.